### DISCUSSION TITLE
Macro support

### DISCUSSION THREAD

**Discussion by matrxbox** [AppGround-io/bluetooth-keyboard-and-mouse-support#210] [Ideas]
I use the text function for some basic macros, but it would be great to have macro specific feaatures like pauses, special keys (ctrl, alt, function keys, mouse buttons, etc)

**Comment by Staini** [comment]
maybe also mouse click coordinates (x/y) ;-)

**Reply by eried** [reply]
I would love this!

**Comment by AppGround-io** [comment]
Version 4.19.0 of the app supports multiple actions for one button with an optional delay. Mouse buttons or movement are not supported yet.
![image](https://user-images.githubusercontent.com/50548429/191682939-04f834c6-dcd2-49c0-9ffb-775d3b4f8cca.png)

**Reply by Staini** [reply]
Great! Thanks a lot.
In a future version, can you please integrate a custom delay field to be entered by the user?
For the case a longer delay is needed.
